modules/DataLoadManager.py

```diff
@@ -157,3 +149,28 @@ def start_single_import(self, target_engine, model_file, requested_full_refresh)
         data_load_tracker.completed_successfully()
         self.data_load_tracker_repository.save(data_load_tracker)
         self.logger.info(f"Import Complete for: {model_name}. {data_load_tracker.get_statistics()}")
+
+    @staticmethod
```

Comment:
Is this method going to be used elsewhere in the future?

Reply:
nope

Reply:
Do we need `@staticmethod`?

Reply:
🤷‍♀️